### PR TITLE
Added jupytercmd and fixed task failure when notify set as true in qubole operator

### LINKED ISSUE #9252: Please update Qubole hook to include Jupyter Notebooks
**Description**

`qds-sdk` already supports JupyterNotebookCommand (https://github.com/qubole/qds-sdk-py/blob/master/qds_sdk/commands.py#L1351) since version 1.15.0 (change notes here https://github.com/qubole/qds-sdk-py/releases/tag/V1.15.0)

and Airflow through `requirements.txt` already includes `qds-sdk` version 1.16.0 https://github.com/apache/airflow/blob/master/requirements/requirements-python3.7.txt#L309

but this hook https://github.com/apache/airflow/blob/master/airflow/providers/qubole/hooks/qubole.py#L26-L29 (also L39-L50) has not been updated yet. 

I suggest we add following the pattern on https://github.com/qubole/qds-sdk-py/blob/master/bin/qds.py#L39

I already planned a fix https://github.com/apache/airflow/commit/785992d1dec6f7f837f1ffbb604535240e22d2bc
but I have never do/opened any PR to Airflow before in my life and I was overwhelmed by the requirements before opening a PR :roll_eyes: (but I understand what the requirements are for) could someone kindly guide me

**Use case / motivation**

I want Airflow's Qubole operator to be able to interact with Qubole Jupyter Notebooks. It's already in the Qubole SDK.

### PR DESCRIPTION
**Pull request by lordozb**
Closes [#9252](https://github.com/apache/airflow/issues/9252)

**Description**
This PR adds jupytercmd in Qubole Operator which fires a JupyterNotebookCommand to the jupyter notebooks running on user's QDS account. Along with this, we have fixed a minor bug which caused the tasks to fail with _--notify_ is set in Qubole Operator.

### CODE REVIEW

**Review by chattarajoy — APPROVED** [review]

**Comment by chattarajoy** [comment]
Upgrade the version of qds sdk to latest in the requirements.

File: airflow/providers/qubole/hooks/qubole.py [hunk 1]
```diff
@@ -239,12 +241,12 @@ def create_cmd_args(self, context):
                     inplace_args = value
                 elif key == 'tags':
                     self._add_tags(tags, value)
+                elif key == 'notify':
+                    if value is True: # pylint: disable=too-many-nested-blocks
```
Comment (chattarajoy) [inline airflow/providers/qubole/hooks/qubole.py]: Add a test case for this?

File: airflow/providers/qubole/operators/qubole.py [hunk 2]
```diff
@@ -106,13 +106,15 @@ class QuboleOperator(BaseOperator):
...
             :sql: inline sql query
             :script_location: s3 location containing query statement
             :language: language of the program, Scala, R, or Python
             :app_id: ID of an Spark job server app
-            :arguments: spark-submit command line arguments
+            :arguments: spark-submit command line arguments.
+                If `cmdline` is selected, this should not be used because all
+                required arguments and configurations are to be passed in the `cmdline` itself.
             :user_program_arguments: arguments that the user program takes in
             :macros: macro values which were used in query
```
Comment (chattarajoy) [inline airflow/providers/qubole/operators/qubole.py:119]: Add parameters for jupytercmd as well

**Review by chattarajoy — CHANGES_REQUESTED** [review]

**Comment by lordozb** [comment]
@chattarajoy 
Master branch already has 1.16.0 version of qds-sdk

**Review by msumit — APPROVED** [review]

**Pull request merged**